src/ui/utils/devtools-toolbox.js

```diff
@@ -37,6 +39,13 @@ export class DevToolsToolbox {
     await this.selectTool(selectedPanel);
   }
 
+  initConsole() {
+    initOutputSyntaxHighlighting();
```

Comment:
We should be able to move this to the JSTerm component for once it mounts so it just takes care of it itself.

https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/webconsole/components/Input/JSTerm.js#L54-L57

Reply:
Also in the same file, line 30-32 we can get rid of this code since don't need to start the console panel anymore.

```
    // The console has to be started immediately on init so that messages appear
    // on the timeline.
    await this.startPanel("console");
```